cephadm-preflight.yml

```diff
@@ -230,8 +230,12 @@
 
         - name: install container engine
           block:
+            - name: verify if podman exists
+              command: apt-cache search '^podman$'
+              register: podman_package
+
```

Comment:
What about using a variable `container_engine` and doing something like following instead:

```yaml
- name: include specific variables
  include_vars: "{{ item }}"
  with_first_found:
    - "{{ ansible_facts['distribution'] }}-{{ ansible_facts['distribution_major_version'] }}.yml"
    - "{{ ansible_facts['os_family'] }}.yml"
 ```

Then you would have only one task

```yaml
- name: install container engine
  package:
    name: "{{ container_engine }}"
    state: present
```

Reply:
This solution is much smarter, give me some time to apply.